.env.example

```diff
@@ -2,7 +2,8 @@ BASE_PATH=./
 BUILD_ROOT_PATH=/app/
 PROJECT_NAME=dhii_containers
 
-PHP_BUILD_VERSION=7.1
-PHP_TEST_VERSION=7.1
+PHP_BUILD_VERSION=7.4
+PHP_TEST_VERSION=8.2
```

Comment:
Yes, that is intentional. With regard to what version we test with, that really depends on what you mean by "test". The `.env` vars are used to set up the local environment, and `PHP_TEST_VERSION` is the version of PHP in the container, where you run your QA tools during development. Consequently, it means that PHPUnit will run on 8.2.

This project supports PHP `^7.4 | & ^8.0`, meaning there are 4 PHP versions to test on. CI [runs on all of them](https://github.com/Dhii/containers/blob/update/.github/workflows/ci.yml#L11-L14), but locally you've got to choose one. Since the project is already built on PHP 7.4, meaning for example that its deps are installed on that version of PHP, and will likely be mostly used by newer PHP versions, it makes sense to test on the latest PHP version to ensure compatibility between the EoL version of the build and the current latest stable of PHP while developing. Once again, though: automation will test on more versions.

Does that address your concern? What are your thoughts?

Reply:
> Does that address your concern? What are your thoughts?
Yes, sure. I'm fine with this, just wanted to make sure this is not a mistake, but a decision. This logic completely makes sense to me, thanks for the explanation!